.pytool/Plugin/LineEndingCheck/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Line Ending Check Plugin
+
+This CiBuildPlugin scans all the files in a package to verify that the line endings are CRLF.
+
+## Configuration
+
+The plugin can be configured to ignore certain files.
+
+``` yaml
+"CharEncodingCheck": {

```

Comment:
this looks incorrect.  Copy and paste error?

Reply:
Yes. Forgot to search the readme. Updated.